integration_tests/src/main/python/orc_test.py

```diff
@@ -377,3 +379,26 @@ def test_read_struct_without_stream(spark_tmp_path):
             lambda spark : unary_op_df(spark, data_gen, 10).write.orc(data_path))
     assert_gpu_and_cpu_are_equal_collect(
             lambda spark : spark.read.orc(data_path))
+
+
+@pytest.mark.parametrize('orc_gen', flattened_orc_gens, ids=idfn)
+@pytest.mark.parametrize('reader_confs', reader_opt_confs, ids=idfn)
+@pytest.mark.parametrize('v1_enabled_list', ["", "orc"])
+def test_read_with_more_columns(spark_tmp_path, orc_gen, reader_confs, v1_enabled_list):
```

Comment:
Spark has the config `spark.sql.caseSensitive` which is used by ORC to decide if columns can be matched up in case sensitive or insensitive way. The default is case insensitive.  Could we have some tests to verify that things like `TOP_AR` and `top_ar` match by default?  Could we also violate the request of Spark and set this to be case sensitive and verify that they do not match as expected?

Reply:
Great idea. 
Found a bug during testing, and fixed it.

Reply:
Updated the names in read schema with some upper chars, it should cover the case-sensitive things.